### PR TITLE
feat: ✨ add .gitignore to exclude unnecessary files

### PR DESCRIPTION
**Pull request by dermatz**

### CODE REVIEW

File: .trunk/.gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+*out
+*logs
+*actions
+*notifications
+*tools
+plugins
+user_trunk.yaml
+user.yaml
+tmp
```
Comment (Morgy93) [inline .trunk/.gitignore:9]: Do we need any of them currently? I'd only add things in case we really need it to keep it small and tidy.

Comment (dermatz) [inline .trunk/.gitignore:9]:
```suggestion
```
remove this entries

Comment (dermatz) [reply .trunk/.gitignore]: You're absolutely right. We don't need that. Thank you.

**Comment by dermatz** [comment]
not needed anymore

**Pull request closed without merging**